anthos-baremetal-edge-deployment/inventory/host_vars/edge-1.yaml

```diff
@@ -0,0 +1,26 @@
+
+# Name of the cluster
+cluster_name: "edge-1"
+# Used for logging/metrics
+machine_label: "{{ cluster_name }}"
+# Indicates that this is the primary machine in the cluster (only important for provisioning)
+primary_cluster_machine: true
+# Name of the "store" per the ACM repo
+acm_cluster_name: "{{ cluster_name }}"
+# This machine's IP
+control_plane_ip: "10.223.62.231"
+# The other IPs in the cluster
+control_plane_ips: [
+  "10.223.62.231",
```

Comment:
Are these the network IPs we want to use?

Reply:
I would suggest we keep with the VXLAN IP CIDR ranges. I went with a 10.200.x.y

Reply:
Done in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/11/commits/d12e8ca4702c0eb4b5172891448b2ac73da9762b